tests/test_admin_client.py

```diff
@@ -307,6 +307,30 @@ async def test_list_subscriptions(
     ]
 
 
+async def test_list_subscriptions_empty_response(
+    app: aiohttp.web.Application,
+    admin_client: Callable[[], Awaitable[AdminClient]],
+    recorder: Recorder,
+) -> None:
+    """Test listing subscriptions."""
+
+    handler = NewHandler(
+        recorder,
+        [{"subscriptions": []}],
```

Comment:
I'm pretty sure its not. (Aside: Usually a best practice is to write the test first, ensure its failing, then add the fix that makes the test pass)

```suggestion
        [{}]
```